tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/CommonJoinExec.java

```diff
@@ -145,7 +145,7 @@ public JoinNode getPlan() {
    * @return True if an input tuple is matched to the left join filter
    */
   protected boolean leftFiltered(Tuple left) {
-    return leftJoinFilter != null && !leftJoinFilter.eval(left).asBool();
+    return leftJoinFilter != null && !leftJoinFilter.eval(left).isTrue();
```

Comment:
No, asBool() incurs an invalid cast exception. Please refer to https://en.wikipedia.org/wiki/Three-valued_logic.